src/seq/quality.jl

```diff
@@ -92,7 +92,7 @@ function infer_quality_encoding(data::Vector{UInt8}, start, stop,
                                 default::QualityEncoding=EMPTY_QUAL_ENCODING)
     @inbounds for i in start:stop
         c = data[i]
-        if '!' <= c <= '~'
+        if UInt8('!') <= c <= UInt8('~')
```

Comment:
I'd like to deprecate this quality encoding inference feature in the next release. So I'd like to keep this as is for now.